client/common.go

```diff
@@ -16,6 +16,10 @@ const (
 	fieldNumSigPubKey = 1
 	fieldNumSigVal    = 2
 	fieldNumSigScheme = 3
+
+	// DefaultTTL is a TTL used for NeoFS requests if no other value is
+	// set specifically for a call.
+	DefaultTTL = 8
```

Comment:
Could you please add the scale of the TTL to the comment? I mean milliseconds, seconds, etc. `8` is quite abstract

Reply:
It's hops! Just a number.